datahub-web-react/src/app/lineage/LineageExplorer.tsx

```diff
@@ -26,10 +28,10 @@ const FooterButtonGroup = styled.div`
     margin: 12px 0;
 `;
 
-const EntityDrawer = styled(Drawer)`
-    top: 106px;
+const EntityDrawer = styled(Drawer)<{ distanceFromTop: number }>`
+    top: ${(props) => props.distanceFromTop}px;
```

Comment:
One other approach is to wrap the path so that it never expands vertically.. in the example shown its a super duplicated long path

Reply:
yeah so the path never actually does expand vertically - it causes a horizontal scroll if it's very long. However once it's long enough, it pushes the "Details", "Lineage", and num upstream and downstream down to the next line which causes things to be weird.

So we could also just make it so that content doesn't get bumped down to a new line, but not sure what that would then look like when the path is long enough to bump into it 🤷